ui/webui/test/check-basic

```diff
@@ -21,7 +21,7 @@ from installer import Installer
 from language import Language
 from review import Review
 from storage import Storage
-from testlib import nondestructive, test_main, wait  # pylint: disable=import-error
+from testlib import nondestructive, test_main, wait, sit  # pylint: disable=import-error
```

Comment:
I don't think you're using this import anywhere?

Reply:
Removed, thank you.